packages/platform-android/src/commands/runAndroid/index.ts

```diff
@@ -46,7 +40,8 @@ export interface Flags {
  * Starts the app on a connected Android emulator or device.
  */
 async function runAndroid(_argv: Array<string>, config: Config, args: Flags) {
-  if (!checkAndroid(args.root)) {
+  const androidProject = config.project.android;
+  if (!androidProject) {
     logger.error(
       'Android project not found. Are you sure this is a React Native project?',
```

Comment:
Not directly related to the PR, but if we did:
```
throw new CLIError(`
  Android project not found. Are you sure this is a React Native project?

  If your Android files are located in a non-standard location (e.g. not inside \'android\' folder), consider setting
  \`project.android.sourceDir\` option to point to a new location.
`)
```
it would be easier to style it later, when we do better error handling.

I would assume that running `run-android` w/o an Android package is most likely an error.